server/routes/main/editor.js

```diff
@@ -6,9 +6,13 @@ var querystring = require("querystring");
 var defaultProjectNameKey = require("../../../constants").DEFAULT_PROJECT_NAME_KEY;
 var utils = require("../utils");
 var HttpError = require("../../lib/http-error");
-
+var env = require("../../lib/environment");
 var snippets = require("../../lib/snippets");
 
+// One of "tarball" or "files" to specify how projects should get loaded.
+// If not specified, the client code will default to "tarball".
+var projectLoadStrategy = env.get("PROJECT_LOAD_STRATEGY");
```

Comment:
can we do this in server/routes/config.js instead and pass in `config.projectLoadStrategy` below?